framework/src/main/java/org/checkerframework/framework/source/messages.properties

```diff
@@ -1,2 +1,3 @@
 annotation.not.completed=Element %s contains annotation %s which can't be found. Make sure your classpath is set correctly.
 unneeded.suppression=warning suppression %s is not used by %s
+type.refinements=variable="%s"; checker="%s"; types=%s
```

Comment:
For the key, just use singular and adapt to the suggestion on the option name.
For the value, in the medium term, I wouldn't limit this to variables. Maybe nothing is needed, as it just gives the type for a particular range.